scripts/apps/workspace/views/workspace-dropdown.html

```diff
@@ -1,3 +1,4 @@
+<h1 id="workspace-heading" class="a11y-only"><span translate>Workspace:</span> {{ selected.name | translate}}</h1>
```

Comment:
> `{{ selected.name | translate}}`

Workspace names are coming from user input so there's no use translating it using strings extracted from the code